services/app/src/components/webFooter/WebFooter.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import './webFooter.css'
+
+const WebFooter = () => (
```

Comment:
Is there any other footer that we render apart from the `Web` footer? If no - rename the component to `Footer`